roles/srv-announcer/meta/main.yml

```diff
@@ -0,0 +1,20 @@
+galaxy_info:
```

Comment:
This role cannot be downloaded by ansible galaxy in its current form. Even with collections, the required folder paths don't match. Therefore, why do you include a galaxy file here?

Reply:
While [my comment below](https://github.com/wireapp/ansible-sft/pull/8#discussion_r478476774) might give some context, you are indeed correct, right now this file has no technical impact, it just holds some, IMHO, useful meta information about the role itself. Would you like me to do anything about this file?

Reply:
Add a comment then, stating that this role cannot be used by ansible galaxy, but that you chose to use the ansible galaxy meta format to describe that this role runs on ubuntu 18... and needs ansible 2.7? It's rather odd.

Why not add this information to the README, and delete this file?

Reply:
The roles can indeed be downloaded via `ansible-galaxy`, as mentioned [here](https://github.com/wireapp/ansible-sft/pull/8#discussion_r480253787).

I don't feel that this odd. The role meta information seem to be the one (and machine-readable) location for such information. At least, it's where I would assume people look first.

> Why not add this information

Those information are already mentioned in the [README.md](https://github.com/wireapp/ansible-sft/blame/add-srv-announcer-role/README.md#L4-L12).